opencti-platform/opencti-front/src/private/components/profile/triggers/TriggerCreation.tsx

```diff
@@ -5,11 +5,15 @@ import SpeedDialIcon from '@mui/material/SpeedDialIcon';
 import SpeedDialAction from '@mui/material/SpeedDialAction';
 import SpeedDial from '@mui/material/SpeedDial';
 import makeStyles from '@mui/styles/makeStyles';
+import { graphql } from 'react-relay';
 import { Theme } from '../../../../components/Theme';
 import { useFormatter } from '../../../../components/i18n';
+import useQueryLoading from '../../../../utils/hooks/useQueryLoading';
 import { TriggersLinesPaginationQuery$variables } from './__generated__/TriggersLinesPaginationQuery.graphql';
 import TriggerDigestCreation from './TriggerDigestCreation';
 import TriggerLiveCreation from './TriggerLiveCreation';
+import NotifierField from '../../common/form/NotifierField';
+import { Option } from '../../common/form/ReferenceField';
```

Comment:
This is a larger issue, we will handle that in the future because we need to be able to pass the filters values to InlineFilters and currently we can't :)